WATOLINK/bci-gaming
# Alexis menu styling

## ClickUp ticket link

[Menu Styling](https://app.clickup.com/t/8685qf8q0)



## Updated the styling to fit the western pixel theme. I transitioned from importing pixel art to using a pixel font to make it easier to add new assets. I updated the main menu styling to have it match the game over screen. 




## Steps to test
1. Play the menu screen and ensure that the models work and the game plays when you click play
2. Switch to the death sequence scene and ensure that the scene plays and the buttons can click without error



## If there is any styling choices I should edit or if anything doesnt work



## Checklist
